src/shared/containers/AddCard.js

```diff
@@ -0,0 +1,35 @@
+import React, {PropTypes} from 'react';
+import { connect } from 'react-redux';
+import { addCard } from '../actions/cardList';
+
+let AddCard = ({ dispatch }) => {
```

Comment:
AddCard is a compoment. Please split it into src/shared/components. AddCardForm would be a good name.